packages/project-disaster-trail/src/components/Game/RadialGauge.js

```diff
@@ -47,13 +34,27 @@ class RadialGauge extends Component {
   };
 
   render() {
-    const { animateGauge, orbSize } = this.props;
+    const { animateGauge, size } = this.props;
     const { percent } = this.state;
 
+    const gaugeDefaultStyle = css`
+      transition: transform 1s;
+      position: absolute;
+      top: ${size / 2}px;
+      left: ${size / 2}px;
+      height: ${size}px;
+      width: ${size}px;
```

Comment:
Thanks for getting that sizing hooked in.

Reply:
You betcha!